src/Resources/skeleton/Admin.tpl.php

```diff
@@ -13,13 +13,13 @@
 final class <?= $class_name ?> extends AbstractAdmin
 {
 
-    protected function configureDatagridFilters(DatagridMapper $datagridMapper): void
+    private function configureDatagridFilters(DatagridMapper $datagridMapper): void
```

Comment:
The function is protected in the AbstractAdmin, shouldn't this stay protected then ?